src/layers/UsePathsLayer.tsx

```diff
@@ -73,6 +78,58 @@ function addUnselectedPathsLayer(map: Map, paths: Path[]) {
     map.addInteraction(select)
 }
 
+function createBezierLineString(start: number[], end: number[]):LineString {
+    var curvePoints = [];
```

Comment:
You can always use `const`, unless you get a (Typescript) compiler error. Then you can use `let`. `var` is basically as thing of the past.

Reply:
done